go.mod

```diff
@@ -2,6 +2,13 @@ module github.com/sigstore/policy-controller
 
 go 1.18
 
+// Bring this in for testing:
```

Comment:
Should we skip adding this ? I assume we don't need it for this code. Right ?

Reply:
haha, yes, was testing something else and these got committed by mistake, thanks for catching this!